GETTING_STARTED.md

```diff
@@ -1,50 +1,50 @@
 ## Getting started
 
-1. Install [Node 16 LTS](https://nodejs.org/en/)
-1. Install [Git](https://git-scm.com/downloads)
+1. Install [Node.js](https://nodejs.org/en/) (LTS version).
+1. Install [Git](https://git-scm.com/downloads).
    - If you are not comfortable with the Git command line tool, you may use GitHub Desktop (<https://desktop.github.com/>). Take a look at <https://acmcsuf.com/1st/> if this is your first time contributing to an open source project.
-1. Create a fork of [acmcsuf.com](https://github.com/EthanThatOneKid/acmcsuf.com), similar to <https://github.com/jaasonw/acmcsuf.com>
-1. Create an [access token](https://scribehow.com/shared/Generating_a_personal_access_token__eMf0UJYaS5OcGrO_lJTpGg)
-1. Follow [these instructions](https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/CONTRIBUTING.md#setting-up-your-env-file) to add it to your dev environment
+1. Create a fork of [acmcsuf.com](https://github.com/EthanThatOneKid/acmcsuf.com), similar to <https://github.com/diamondburned/acmcsuf.com>
```

Comment:
You added a period at the end of every bullet point, did you want to add one here too?

Reply:
Since these are phrased like complete sentences, I agree with you that a period is appropriate.

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865/commits/24e54d0e8a43b101bdc2a973add4ec1206957805>.